### PR TITLE
Up to version v0.4.0 

### PR DESCRIPTION
**Pull request by mprevide**

### CODE REVIEW

File: source/releases.rst [hunk 1]
```diff
@@ -1,56 +1,79 @@
...
-    They are split into four different access levels: tenant, flow, node and node
-    instance. Check `flowbroker node library <https://github.com/dojot/flowbroker/blob/master/lib/ContextHandler.js>`_
-    to check how to use context within nodes or check `flowbroker's get-context node <https://github.com/dojot/flowbroker/tree/master/orchestrator/nodes/get-context>`_
-    to use it directly from flowbroker GUI  you could just open the new
-    flowbroker UI and check it in the node palette)
-  - New configuration options for device actuation: send actuation message to the
-    same device that triggered the flow or set which is the targed device dynamically,
-    set while the flow is being processed.
-  - Support for device information caching (improving performance)
+  - New node Event Device In
```
Comment (raulnegreiros) [inline source/releases.rst]: do we need to add the remotion of the email node?

File: source/locale/pt_BR/LC_MESSAGES/concepts.po [hunk 2]
```diff
@@ -25,7 +25,9 @@ msgstr "Conceitos"
 msgid ""
 "This document provides information about dojot's concepts and "
 "abstractions."
-msgstr "Este documento fornece informações sobre os conceitos e abstrações do dojot."
+msgstr ""
+"Este documento fornece informações sobre os conceitos e abstrações do "
```
Comment (raulnegreiros) [inline source/locale/pt_BR/LC_MESSAGES/concepts.po]: I think it is necessary to uniform dojot's gender...

File: source/installation-guide.rst [hunk 3]
```diff
@@ -155,7 +153,7 @@ The following sections describe all dependencies and steps required
 for this deployment.
 
 .. attention::
-  At this moment, we only support Kubernetes in version 1.15 with dojot v0.3.1 (battojutsu.1)
+  At this moment, we only support Kubernetes util version 1.15 with dojot v0.3.1 (battojutsu.1)
```
Comment (raulnegreiros) [inline source/installation-guide.rst]: typo?

File: source/installation-guide.rst [hunk 4]
```diff
@@ -171,7 +169,7 @@ documentation`_.
 Kubernetes Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-- The Kubernetes supported version is **v1.11**.
+- The Kubernetes supported version is between **v1.11** and **v1.15** .
```
Comment (raulnegreiros) [inline source/installation-guide.rst]: Why do we support these specific versions only? It sounds a bit strange, in a previous paragraph, you have said another requirement.

File: source/locale/pt_BR/LC_MESSAGES/releases.po [hunk 5]
```diff
@@ -85,107 +84,110 @@ msgstr ""
...
 #: ../../source/releases.rst:32
-#, fuzzy
 msgid "New node Notification"
-msgstr "Notificações"
+msgstr "Novo nó de Notificações"
 
 #: ../../source/releases.rst:33
 msgid "New node Multi Actuate"
-msgstr ""
+msgstr "Novo nó de multiplias atuações (Multi Actuate)"
```
Comment (raulnegreiros) [inline source/locale/pt_BR/LC_MESSAGES/releases.po]: "Novo nó de atuação para vários dispositivos (Multi Actuate)"
just to keep the standard

File: source/locale/pt_BR/LC_MESSAGES/releases.po [hunk 6]
```diff
@@ -85,107 +84,110 @@ msgstr ""
...
 
-#: ../../source/releases.rst:47
+#: ../../source/releases.rst:48
 msgid "Improvements to support Firmware Update"
 msgstr "Melhorias para dar suporte à atualização de firmware"
 
-#: ../../source/releases.rst:49
+#: ../../source/releases.rst:50
 msgid "DataBroker:"
 msgstr ""
```
Comment (raulnegreiros) [inline source/locale/pt_BR/LC_MESSAGES/releases.po]: missing?

File: source/locale/pt_BR/LC_MESSAGES/releases.po [hunk 7]
```diff
@@ -85,107 +84,110 @@ msgstr ""
...
 
-#: ../../source/releases.rst:51
+#: ../../source/releases.rst:52
 msgid "Support notifications in socket-io"
 msgstr "Suporte a Notificações no socket-io"
 
-#: ../../source/releases.rst:53
+#: ../../source/releases.rst:54
 msgid "DataManager:"
 msgstr ""
```
Comment (raulnegreiros) [inline source/locale/pt_BR/LC_MESSAGES/releases.po]: missing?

File: source/locale/pt_BR/LC_MESSAGES/releases.po (hunk 6 above)
Comment (mprevide) [reply source/locale/pt_BR/LC_MESSAGES/releases.po]: No, it just doesn't have to be translated.

**Pull request merged**